.github/workflows/playground-deployment-preview.yml

```diff
@@ -0,0 +1,23 @@
+name: Playground application deployment preview trigger
+
+on:
+  workflow_dispatch:
+  repository_dispatch:
+    types: [vercel_deployment]
```

Comment:
Just to understand: this workflow is expected to be triggered manually?

Reply:
Once the project is setup, we will set a webhook in Vercel to our repository which will be called whenever a deployment is done.

The idea is that we will use that webhook to trigger this gh action in order to create/update the deployment preview in the Merchant Center.

That's something we can't do now (because of the missing auth token) but I wanted to set it up and verify it works so it would be easier for us in the future to integrate it once the auth token is available.

![image](https://github.com/commercetools/merchant-center-application-kit/assets/97907068/a6afd4c3-6499-43dd-991e-d0c26c6449c4)

Reply:
Thanks, this makes sense now.